src/libs/ReportUtils.ts

```diff
@@ -3935,33 +3843,41 @@ function shouldDisableRename(report, policy) {
 
     // If there is a linked workspace, that means the user is a member of the workspace the report is in.
     // Still, we only want policy owners and admins to be able to modify the name.
-    return !_.keys(loginList).includes(policy.owner) && policy.role !== CONST.POLICY.ROLE.ADMIN;
+    return !Object.keys(loginList ?? {}).includes(policy.owner) && policy.role !== CONST.POLICY.ROLE.ADMIN;
 }
 
+type OnyxDataTaskAssigneeChat = {
+    optimisticData: OnyxUpdate[];
+    successData: OnyxUpdate[];
+    failureData: OnyxUpdate[];
+    optimisticAssigneeAddComment?: OptimisticReportAction;
+    optimisticChatCreatedReportAction?: OptimisticCreatedReportAction;
+};
+
 /**
  * Returns the onyx data needed for the task assignee chat
- * @param {Number} accountID
- * @param {Number} assigneeAccountID
- * @param {String} taskReportID
- * @param {String} assigneeChatReportID
- * @param {String} parentReportID
- * @param {String} title
- * @param {Object} assigneeChatReport
- * @returns {Object}
- */
-function getTaskAssigneeChatOnyxData(accountID, assigneeAccountID, taskReportID, assigneeChatReportID, parentReportID, title, assigneeChatReport) {
+ */
+function getTaskAssigneeChatOnyxData(
+    accountID: number,
+    assigneeAccountID: number,
+    taskReportID: string,
+    assigneeChatReportID: string,
+    parentReportID: string,
+    title: string,
+    assigneeChatReport: OnyxEntry<Report>,
+): OnyxDataTaskAssigneeChat {
     // Set if we need to add a comment to the assignee chat notifying them that they have been assigned a task
     let optimisticAssigneeAddComment;
     // Set if this is a new chat that needs to be created for the assignee
     let optimisticChatCreatedReportAction;
```

Comment:
Add types for these variables

Reply:
not sure why but when I added types to those variables I am getting ts error 🤔 
<img width="773" alt="image" src="https://github.com/Expensify/App/assets/25230417/6ccd7f6f-e301-40d6-bc41-7c54a6a487af">

Do you know why ?